networkmanager/netpool.go

```diff
@@ -87,7 +92,7 @@ func makeNetPool(size int, base *net.IPNet) (listIPNet []*net.IPNet) {
 
 func addIntToIP(array net.IP, ordinal uint) {
 	for i := len(array) - 1; i >= 0; i-- {
-		array[i] |= (byte)(ordinal & 0xff)
+		array[i] |= (byte)(ordinal & 0xff) // nolint:gomnd
```

Comment:
Add it as exception to linter settings instead of suppressing in the code.

Reply:
Done